eventbridge/README.md

```diff
@@ -0,0 +1,45 @@
+# eventbridge
+
+A Wing library for working with [Amazon EventBridge](https://aws.amazon.com/eventbridge/).
+
+## Prerequisites
+
+* [winglang](https://winglang.io)
+
+## Installation
+
+```sh
+npm i @winglibs/eventbridge
+```
+
+## Usage
+
+```js
+bring eventbridge;
+
+let bus = new eventbridge.Bus();
+
+bus.subscribeFunction("github.pull-request.created", inflight (event) => {
```

Comment:
Kind of feels like `subscribeFunction` should take a `cloud.Function` and this should be called `onEvent`

Reply:
Not required